package.json

```diff
@@ -53,9 +53,12 @@
     "syncpack": "^13.0.0",
     "ts-node": "^10.9.2",
     "tslib": "^2.7.0",
-    "tsx": "^4.19.1",
     "tsup": "^8.3.0",
+    "tsx": "^4.19.1",
     "typescript": "^5.6.2",
     "zx": "8.1.8"
+  },
+  "dependencies": {
+    "csv-stringify": "6.5.1"
```

Comment:
this repository acts as a monorepo.
please install this package directly on the relevant internal package that uses it.

Reply:
@davidmeirlevy this is currently installed in the lib package. Must it be installed in a different package?

Reply:
@ovindu-a i guess @davidmeirlevy ment the `@configu/lib` package